src/transforms/regex_parser.rs

```diff
@@ -232,27 +275,12 @@ impl Transform for RegexParser {
                     }
                 }
 
-                for (idx, name, conversion) in &self.capture_names {
-                    if let Some((start, end)) = capture_locs.get(*idx) {
-                        let capture: Value = value[start..end].into();
-                        match conversion.convert(capture) {
-                            Ok(value) => {
-                                let name = match &self.target_field {
-                                    Some(target) => Atom::from(format!("{}.{}", target, name)),
-                                    None => name.clone(),
-                                };
-                                log.insert(name, value);
-                            }
-                            Err(error) => {
-                                debug!(
-                                    message = "Could not convert types.",
-                                    name = &name[..],
-                                    %error,
-                                    rate_limit_secs = 30
-                                );
-                            }
-                        }
-                    }
+                for (name, value) in captures {
+                    let name = match &self.target_field {
+                        Some(target) => Atom::from(format!("{}.{}", target, name)),
+                        None => name.clone(),
+                    };
```

Comment:
Consider the following, might be a matter of taste for functional mapping:
```rust
(&self.target_field).map(|target| Atom::from(format!("{}.{}", target, name)).unwrap_or_else(|| name.clone());
```

Reply:
:+1: This was just moved, but I like yours better.